### ISSUE TITLE
Fail: install and enable vscode-smoketest-check extension

### ISSUE THREAD

**Issue by jrieken** [microsoft/vscode#139183]
install and enable vscode-smoketest-check extension

**Comment by jrieken** [comment]
test is skipped

**Comment by sandy081** [comment]
@jrieken  just want to check if the link in description is working? Its opening a blank page for me.

**Comment by sandy081** [comment]
I can open it now. Seems like a network trouble.

**Comment by sandy081** [comment]
From the following image, it seems the installation of extension is not completed within timeout time.

![image](https://user-images.githubusercontent.com/10746682/146240429-9da64e7a-0909-4d66-9a28-7fc1d2c89517.png)

Server logs do not have info about extension installation and also the window log. It could be network issue too.

@bpasero Is it possible to run smoke tests always in verbose mode?

@Tyriar I do not think of any action can be taken here unless we have verbose logs. This test is disabled as of now. What is recommended in this case?

**Comment by bpasero** [comment]
I can look into enabling verbose logs, maybe at least for as long as we see this flakiness.

**Comment by Tyriar** [comment]
I setup luna paint to retry once and enable logs when it retries which works nicely, should we do the same?

**Comment by bpasero** [comment]
I don't think there is any harm in always running with verbose logs, in the end we need all the data we can find. The only downside is that we would then run smoke tests in a different mode than our normal users, not sure that would have an impact.

**Comment by sandy081** [comment]
Shall I enable this smoke test to get more information with logs?

**Comment by bpasero** [comment]
Yeah, I didn't get to do the verbose logging, I suggest you enable the test and add some logging as `info` for the time being.

**Comment by sandy081** [comment]
Did not change all trace statement to info, but added one log statement expecting to get some info. 

It would be great if verbose is enabled to get all other info.

**Comment by JacksonKearl** [comment]
https://dev.azure.com/monacotools/Monaco/_build/results?buildId=155054&view=logs&j=963754b8-394e-5caf-de5d-2790b068caec&t=636c104c-b909-5cac-59f5-95c1f9c594ad
